docs/static/open-in-gitpod.svg

```diff
@@ -1,23 +0,0 @@
-<svg width="160" height="45" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These images are being used in some documentations?

Reply:
Yes, but they already exist in the docs repo.
https://gitea.com/gitea/gitea-docusaurus/src/branch/main/static

Reply:
Okay, I'd re-added them here and removed them in the docs repo PR.